scaffold/phpstan.neon

```diff
@@ -1,2 +1,13 @@
 includes:
-	- vendor/lullabot/drainpipe-dev/config/phpstan.neon
+    - ../../../mglaman/phpstan-drupal/extension.neon
+    - ../../../phpstan/phpstan-deprecation-rules/rules.neon
```

Comment:
@justafish what's the reason for changing these? Will the previously shipped phpstan.neon be removed in drainpipe 4? Wondering what your recommendations are for sites that have customized this file.